### PR TITLE
Toolbar and omnibar changes

### PR DESCRIPTION
**Pull request by MasterK0927**
1) Hid the Brave_action icons i.e. brave_shield and brave_rewards.
2) Hid the Licensed photo author toggle from the background.
3) Stopped the generation of License for the background image.

### CODE REVIEW

File: script/generate_licenses.py [hunk 1]
```diff
@@ -161,36 +161,36 @@ def validated_data_field(data, field_name):
     return field_value
 
 
-def generate_backgrounds_license(preamble, backgrounds):
```
Comment (udaybansal19) [inline script/generate_licenses.py]: we dont need to comment this out if already commented below

File: script/generate_licenses.py [hunk 2]
```diff
@@ -233,12 +233,12 @@ def main():
         'background images used on the new tab page. The Brave Browser and ' \
         'such data files are separate and independent works.'
 
-    ntp_backgrounds = list_ntp_backgrounds(
-        os.path.join(ntp_data_dir, 'backgrounds.ts'))
-    if write_license_file(ntp_data_dir, generate_backgrounds_license(
-            ntp_backgrounds_preamble, ntp_backgrounds)):
-        print(f'- {len(ntp_backgrounds)} sub-components added in ' \
-              'brave_new_tab_ui/data/LICENSE')
+    # ntp_backgrounds = list_ntp_backgrounds(
```
Comment (udaybansal19) [inline script/generate_licenses.py:236]: try to remove only the things not needed. if this is already removed from react then need not comment this

File: script/generate_licenses.py (hunk 1 above)
Comment (MasterK0927) [reply script/generate_licenses.py]: actually yes, but this code is not needed by us

File: components/brave_shields/resources/panel/brave_shields.html [hunk 3]
```diff
@@ -1,4 +1,4 @@
-<!doctype html>
+<!-- <!doctype html>
```
Comment (udaybansal19) [inline components/brave_shields/resources/panel/brave_shields.html:1]: what is this file doing?

**Pull request closed without merging**